### PR TITLE
Feature/fe 18 profile details page

### PR DESCRIPTION
**Pull request by aaman123**
### What this PR does (required):
- Profile Details Page

### Screenshots / Videos (required):
- 
Screenshot (1563)

### Any information needed to test this feature (required):
- Nope

### Any issues with the current functionality (optional):
- Nope

### CODE REVIEW

**Review by ugur-akin — COMMENTED** [review]
Couldn't see a mock for the design but what you did looks really nice. I commented on a couple of things that could be improved. Also an idea that might be worth considering is to make the availability form `sticky` so that it's visible even after we scroll down to browse a longer profile page.

File: client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                LOGOUT="/logout"
+                PROFILE="/profile"
+                MY_SITTER="/mySitter"
+                BECOME_SITTER="/becomeSitter"
+                MESSAGE="/message"
+                Logout="Log out"
+                Profile="Profile"
+                Mysitter="My Sitters"
+                Becomesitter="BECOME A SITTER"
+                Messages="Messages"
```
[the remark this answers is not in the document]
Comment (aaman123) [reply client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx:27]: hii ugur, I think this Navbar component is created by Ibsa ...

[the remark this answers is not in the document]
Comment (rajivtitus) [reply client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx:27]: Hey @aaman123 and @ibsa-ejeta, can either of you please make a quick fix here?  Thanks :)

File: client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                                <Typography gutterBottom align="center" variant="h4">$14/hr</Typography>
+                                <Typography align="center">
+                                    <Rating name="simple-controlled" />
+                                </Typography>
+                                {/* HANDLING FORMDATA YET TO BE IMPLEMENTED */}
+                                <form>
+                                    <Box p={4}>
+                                        <Box>    
+                                            <Typography variant="caption" align="left">DROP IN</Typography>
+                                            <TextField id="datetime-local" type="datetime-local" className={classes.dateInput} />
```
Comment (rajivtitus) [inline client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx:70]: The id for the component is supposed to be a unique field, please change it here or in the component below.

File: client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                            </Card>
+                        </Paper>
+                    </Grid>
+
+                    <Grid item>
+                        <Paper elevation={12} className={classes.availWrapper}>
+                            <Box p={5}>
+                                <Typography gutterBottom align="center" variant="h4">$14/hr</Typography>
+                                <Typography align="center">
+                                    <Rating name="simple-controlled" />
```
Comment (rajivtitus) [inline client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx:63]: Rating is supposed to be a read only component here

**Review by rajivtitus — APPROVED** [review]
Good work on the UI! Just some minor comments, please address before merging :)

**Pull request merged**